test/namespaces.bats

```diff
@@ -36,6 +36,9 @@ function teardown() {
 }
 
 @test "pid namespace mode target test" {
+	if [[ -z "$CONTAINER_UID_MAPPINGS" || $(id -u) == "0" ]]; then
+		skip "test fails in a user namespace"
+	fi
```

Comment:
The check seems to skip the test if there's NO userns enabled (as well as when the user is root), but the message suggests otherwise. I guess I'm missing something here 😕